plugin-build/build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.4.32"
```

Comment:
maybe we move all the names and versions to `BuildPluginsVersion`?

Reply:
Yup. I had to share the `buildSrc` between the two builds are they're isolated by default.